docs/templates/README.md

```diff
@@ -0,0 +1,72 @@
+# FinOps toolkit templates
+
+The FinOps toolkit uses [Azure Resource Manager (ARM) templates](https://learn.microsoft.com/azure/azure-resource-manager/templates/). Our templates use [Bicep modules](./modules) for reuse across templates.
+
+Please refer to [Creating templates](#Creating-templates) below for template requirements.
+
+Templates:
+
+- [finops-hub](./finops-hub.md)
+- [finops-hub-with-exports](./finops-hub-with-exports.md)
+
+---
+
+## Creating templates
+
+Templates follow the [Azure Quickstart Templates guidelines](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md#contribution-guide):
+
+1. Every template should be in its own folder.
```

Comment:
Aren't we breaking this by having the two templates in the root templates folder?

Reply:
This is just the docs. I need to move this over to dev docs under src since these notes are more internal than external. I'll do that in a future PR.